### DISCUSSION TITLE
securing an API

### DISCUSSION THREAD

**Discussion by SySafarila** [spatie/laravel-permission#2205] [Q&A]
Hello, can i use this laravel-permission package to secure my api ? because i tried using middleware like : `Route::get('/test', function() {
        return response()->json('testing');
    })->middleware('auth:api', 'can:admin access');` its returned unauthorized response, but if i use `middleware('auth:api', 'role:admin')` its returned "testing" and this is what i want, but on my web.php i usually use the `can:xxx` to secure the routes and this is not work in api.php

**Comment by erikn69** [comment]
Can is working, look at the tests
